lib/state/selectors.ts

```diff
@@ -1,6 +1,7 @@
 import * as S from './';
 import * as T from '../types';
 import isEmailTag from '../utils/is-email-tag';
+import { tagHashOf, tagNameOf } from '../utils/tag-hash';
```

Comment:
uh oh, looks like `tagNameOf` isn't in use here

Reply:
Oops, I forgot to remove this 🙇  .

Reply:
Fixed in this [commit](https://github.com/Automattic/simplenote-electron/pull/2817/commits/14e07f06cd5c765b8cfc0d1baa3bd8541ed8895a).